src/System.Data.SqlClient/tests/FunctionalTests/DiagnosticTest.cs

```diff
@@ -0,0 +1,507 @@
+﻿using Xunit;
+using System.Reflection;
+using System.Diagnostics;
+using System.Collections;
+using System.Xml;
+using Xunit.Abstractions;
+using System.Threading.Tasks;
+
+namespace System.Data.SqlClient.Tests
+{
+    public class DiagnosticTest
+    {
+        private string _connectionString = "server=tcp:server,1432;database=test;uid=admin;pwd=SQLDB;connect timeout=60;";
+        private readonly ITestOutputHelper _output;
+
+        public DiagnosticTest(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        [Fact]
+        private void ExecuteScalarTest()
```

Comment:
Ok, thanks.  FWIW, for consistency with the rest of the repo, I'd suggest making the [Fact] methods public.  Otherwise I believe these will be the only ones in the whole repo that are private ;)